pkg/network/ovn_kubernetes.go

```diff
@@ -219,13 +240,21 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		}
 	}
 	renderOVNFlowsConfig(bootstrapResult, &data)
-	if len(bootstrapResult.OVN.MasterIPs) == 1 {
+	if len(bootstrapResult.OVN.MasterAddresses) == 1 {
 		data.Data["IsSNO"] = true
 	} else {
 		data.Data["IsSNO"] = false
 	}
 
-	manifests, err := render.RenderDir(filepath.Join(manifestDir, "network/ovn-kubernetes"), &data)
+	manifestDirs := make([]string, 2)
```

Comment:
minor nit: I prefer `make([]string, 0, 2)`

Reply:
Do you mean `make([]string, 2, 2)` ?

Reply:
Either `make([]string, 0, 2)` nor `make([]string, 2, 2)` works.
but `make([]string, 2)` works.

Reply:
```
pkg/network/ovn_kubernetes.go:246:33: S1019: should use make([]string, 2) instead (gosimple)
	manifestDirs := make([]string, 2, 2)
	                               ^
level=info msg="File cache stats: 1 entries of total size 43.4KiB"
```

Reply:
right, in this case it's fine, but `make([]string, 0, 2)` and then `append(...)` has the minor advantage that if one of the `append`s is commented out, we don't wind up with an array of empty strings.

Just personal preference, and not important here.